### PR TITLE
2517 concent/force report integration tests

### PR DESCRIPTION
**Pull request by shadeofblue**
not quite yet completely addresses #2517

### CODE REVIEW

File: scripts/concent_integration_tests/basic/test_basic.py [hunk 1]
```diff
@@ -18,20 +18,20 @@ def test_send(self):
 
         logger.debug("Sending FRCT: %s", msg)
 
-        response = self._send_to_concent(msg)
+        response = self.send_to_concent(msg)
```
Comment (jiivan) [inline scripts/concent_integration_tests/basic/test_basic.py]: :+1:

File: scripts/concent_integration_tests/force_report/test_messages.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        frct_rcv = self.requestor_receive()
+
+        rrct = message.tasks.RejectReportComputedTask(
+            task_to_compute=frct_rcv.report_computed_task.task_to_compute,
+            reason=message.tasks.RejectReportComputedTask.
+                REASON.SubtaskTimeLimitExceeded
+        )
+
+        response = self.requestor_send(rrct)
+        raise Exception(response)  # @todo need to figure out what exactly happens here
```
Comment (jiivan) [inline scripts/concent_integration_tests/force_report/test_messages.py]: hmmm :?

File: tests/golem/network/p2p/test_peersession.py [hunk 3]
```diff
@@ -516,7 +516,7 @@ def _gen_data_for_test_react_to_remove_task(self):
 
         task_id = "test_{}".format(uuid.uuid4())
         msg = message.RemoveTask(task_id=task_id)
-        msg.serialize(sign_func=keys_auth.sign)
+        msg.serialize(keys_auth._private_key)
```
Comment (jiivan) [inline tests/golem/network/p2p/test_peersession.py]: explicit keyword argument would be more resistant to extending method signature in the future

File: tests/golem/task/test_tasksession.py [hunk 4]
```diff
@@ -622,7 +622,7 @@ def test_get_resource(self):
 
         sess = TaskSession(conn)
         sess.send = lambda m: db.append_bytes(
-            m.serialize(lambda x: b'\000' * 65),
+            m.serialize(None),
```
Comment (jiivan) [inline tests/golem/task/test_tasksession.py]: just `.serialize()`?

Comment (shadeofblue) [reply tests/golem/task/test_tasksession.py]: ah, right :)

**Review by jiivan — APPROVED** [review]

**Pull request merged**